cms/static/sass/views/_settings.scss

```diff
@@ -226,6 +226,16 @@
           }
         }
 
+        .input-minimum-grade {
+          float: left;
```

Comment:
please use float method used in https://github.com/edx/edx-platform/blob/master/lms/static/sass/vendor/bi-app/_mixins.scss#L77